packages/headless/package.json

```diff
@@ -71,6 +71,8 @@
     "@rollup/plugin-replace": "2.4.2",
     "@types/jest": "27.0.3",
     "concurrently": "6.4.0",
+    "esbuild": "^0.14.1",
```

Comment:
Esbuild build is still `0.x`. Additional information around [production readiness](https://esbuild.github.io/faq/#production-readiness) available here.

Reply:
I'm fine with that, in this case.